rocAL/rocAL_pybind/examples/rocAL_api_python_unittest.py

```diff
@@ -52,154 +73,290 @@ def main():
     rocal_cpu = False if args.rocal_gpu else True
     device = "cpu" if rocal_cpu else "cuda"
     batch_size = args.batch_size
+    max_height = args.max_height
+    max_width = args.max_width
+    color_format = types.RGB if args.color_format else types.GRAY
+    tensor_layout = types.NHWC if args.color_format else types.NCHW
+    tensor_dtype = types.UINT8
     num_threads = args.num_threads
     random_seed = args.seed
-    local_rank =  args.local_rank
-    world_size =  args.world_size
-    display = True if args.display else False
+    local_rank = args.local_rank
+    world_size = args.world_size
+    interpolation_type = INTERPOLATION_TYPES[args.resize_interpolation_type]
+    scaling_mode = SCALING_MODES[args.resize_scaling_mode]
+    if (scaling_mode != types.SCALING_MODE_DEFAULT and interpolation_type !=
+            types.LINEAR_INTERPOLATION):
+        interpolation_type = types.LINEAR_INTERPOLATION
+    if augmentation_name in ["hue", "saturation", "color_twist"] and color_format == types.GRAY:
+        print("Not a valid option! Exiting!")
+        sys.exit(0)
+
     try:
-        path= "OUTPUT_IMAGES_PYTHON/NEW_API/FILE_READER/" + args.augmentation_name
+        path = "OUTPUT_IMAGES_PYTHON/NEW_API/FILE_READER/" + args.augmentation_name
```

Comment:
@fiona-gladwin I have mentioned this before. Please remove New_api from all tests. It doesn't make sense to anyone outside

Reply:
Removed the folder name from the python unit tests